example/settlement/main.go

```diff
@@ -226,7 +226,7 @@ func main() {
 		NextStop:      "HKTCSL",
 	}
 
-	if blk, err := client.Settlement.GetProcessCDRBlock(&pccwAddr, cdr1, func(hash types.Hash) (signature types.Signature, err error) {
+	if blk, err := client.Settlement.GetProcessCDRBlock(&pccwAddr, []*qlcchain.CDRParam{cdr1}, func(hash types.Hash) (signature types.Signature, err error) {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
229-252 lines are duplicate of `example/settlement/main.go:265-288` (dupl)